### PR TITLE
create LandingPage and patch GasStation component

### PR DESCRIPTION
**Pull request by D3nnis249**
Thank you for your review :),

We have written a landing page and a component for the different stations

-client/src/components/GasStation.jsx
-client/src/pages/LandingPage.jsx

### CODE REVIEW

**Comment by D3nnis249** [comment]
we have decided not to use vercel for the time being because it can happen from time to time that the app has errors. later on we will use vercel.

File: client/src/components/GasStation.jsx [hunk 1]
```diff
@@ -1,37 +1,43 @@
...
-        </li>
-      )}
-    </>
+    // <StyledMain>
+    <GasStationList role="list">
+      {data.map(station => (
+        <GasStationItem className="ListItems" key={station.uuid}>
+          <p>{station.e5}</p>
+          <h2>{station.brand}</h2>
+          <h3>{station.name.length > 14 ? `${station.name.substring(0, 14)}...` : station.name}</h3>
```
Comment (PReichetanz) [inline client/src/components/GasStation.jsx]: I don't understand semantics here: Why is there a p element above h2 and h3?

File: client/src/components/GasStation.jsx [hunk 2]
```diff
@@ -1,37 +1,43 @@
...
-          <h2>{name}</h2>
-        </li>
-      ) : (
-        <li onClick={handleDetails}>
-          <h2>{name}</h2>
-          <h2>{e5}</h2>
-        </li>
-      )}
-    </>
+    // <StyledMain>
```
Comment (PReichetanz) [inline client/src/components/GasStation.jsx]:
```suggestion
```

File: client/src/components/GasStation.jsx [hunk 3]
```diff
@@ -1,37 +1,43 @@
...
+          <h2>{station.brand}</h2>
+          <h3>{station.name.length > 14 ? `${station.name.substring(0, 14)}...` : station.name}</h3>
+          <p>{station.street}</p>
+          <p>{station.post_code}</p>
+          <p>{station.city}</p>
+          <img src={FavoriteIconDeactivated} alt="Favorite icon deactivated" />
+        </GasStationItem>
+      ))}
+    </GasStationList>
+    // </StyledMain>
```
Comment (PReichetanz) [inline client/src/components/GasStation.jsx]:
```suggestion
```

**Review by PReichetanz — COMMENTED** [review]
Nice work, only an issue regarding semantics 🚀

**Review by PReichetanz — APPROVED** [review]
Looks good to me!

**Pull request merged**